x-pack/plugin/data-streams/src/main/java/org/elasticsearch/xpack/datastreams/DataStreamsPlugin.java

```diff
@@ -48,6 +61,45 @@
 
 public class DataStreamsPlugin extends Plugin implements ActionPlugin {
 
+    public static final Setting<TimeValue> TIME_SERIES_POLL_INTERVAL = Setting.timeSetting(
+        "time_series.poll_interval",
+        TimeValue.timeValueMinutes(1),
+        TimeValue.timeValueMinutes(1),
+        TimeValue.timeValueMinutes(10),
```

Comment:
Out of curiosity, where did we get these numbers? We originally thought about tying it to the ILM poll interval, which is 10m by default (I understand we don't want to tie it to that), so I am curious how we got to 1m < default of 1m < 10m for the value?

Reply:
I should have set the default to `5m`. The valid range of values (1m to 10m) is based on that I like to run this often. 
This interval shouldn't be larger than `index.look_ahead_time`. The minus allowed value for this setting is `2h`. So we could allow for a higher `time_series.poll_interval` value here.